tests/unit/test_dbt_runner.py

```diff
@@ -5,7 +5,7 @@
 
 class TestDbtRunner:
     @pytest.fixture
-    def dbt(self) -> dbtRunner:
```

Comment:
This feels like gonna introduce some overhead of setting up the project to unittests?